.github/workflows/r_nightly.yml

```diff
@@ -60,7 +60,7 @@ jobs:
           repository: ursacomputing/crossbow
           ref: main
       - name: Set up Python
-        uses: actions/setup-python@42375524e23c412d93fb67b49958b491fce71c38 # v5.4.0
+        uses: actions/setup-python@42375524e23c412d93fb67b49958b491fce71c38 # v5.4.0

```

Comment:
I don't recall editing this, but perhaps the line ending characters have been changed (there was a warning about mixing LF and CRLF when opened on GitHub. I did the edit in VSCode on GitHub, which probably automatically unified the end-of-line characters).

Reply:
Right. We should unify end-of-line characters.